tests/test_run.py

```diff
@@ -206,5 +208,100 @@ def test_filepath_dataset_run(generate_local_dataset, images_per_class):
     imagelab = Imagelab(filepaths=filepaths)
     imagelab.find_issues()
     imagelab.report()
-    assert len(imagelab.issues.columns) == 14
+    assert len(imagelab.issues.columns) == 16
     assert len(imagelab.issues) == images_per_class
+
+
+@pytest.mark.usefixtures("set_plt_show")
+def test_filepath_dataset_size_negative(generate_local_dataset_once, images_per_class):
+    """
+
+    All images are same size, so no image should have an size issue
+
+    """
+    files = os.listdir(generate_local_dataset_once / "class_0")
+    filepaths = [
+        os.path.join(generate_local_dataset_once / "class_0", f) for f in files
+    ]
+    imagelab = Imagelab(filepaths=filepaths)
+    imagelab.find_issues()
+    imagelab.report()
+    assert len(imagelab.issues.columns) == 16
+    assert len(imagelab.issues[imagelab.issues["is_size_issue"] == True]) == 0
+
+
+@pytest.mark.usefixtures("set_plt_show")
+def test_filepath_dataset_size_to_large(generate_local_dataset_once, images_per_class):
+    """
+    Size issue is defined based on the area of an image. If the area (width * height) is larger than the median
+    area*threshold(default 10),is_size_issue is set to True. In this example, the median area is 300x300 so 90000.
+    An image with 950 x 950 has an area  of 902500 so its more than 10x larger and thus should be flagged.
+    """
+    arr = np.random.randint(low=0, high=256, size=(950, 950, 3), dtype=np.uint8)
+    img = Image.fromarray(arr, mode="RGB")
+    img.save(Path(generate_local_dataset_once / "class_0" / "larger.png"))
+
+    files = os.listdir(generate_local_dataset_once / "class_0")
+    filepaths = [
+        os.path.join(generate_local_dataset_once / "class_0", f) for f in files
+    ]
+    imagelab = Imagelab(filepaths=filepaths)
+    imagelab.find_issues()
+    imagelab.report()
```

Comment:
you could get rid of `imagelab.report()` in all the tests, as it's not being used anywhere